packages/tests/tests/filter/subscribe.node.spec.ts

```diff
@@ -152,7 +164,7 @@ const runTests = (strictCheckNodes: boolean): void => {
 
       // Send a test message using the relay post method.
       const relayMessage = ServiceNodesFleet.toMessageRpcQuery({
-        contentTopic: TestContentTopic,
```

Comment:
I would do this tidy up in a separate PR to focus this PR on light push v3

Reply:
this was actually helping the tests not fail for me

Reply:
Ah, but was it because it make the test more stable, or because of your changes?